sender-kinesis/src/main/java/zipkin/reporter/kinesis/KinesisSender.java

```diff
@@ -68,6 +69,9 @@ public static Builder builder() {
     /** Maximum size of a message. Kinesis max message size is 1MB */
     Builder messageMaxBytes(int messageMaxBytes);
 
+    /** Allows you to change to json format. Default is thrift */
+    Builder encoding(Encoding encoding);
```

Comment:
On second thought, Encoder.JSON2 doesn't make sense until we have a Span2 type, particularly as Span(1) can have multiple endpoints and that doesn't translate neatly.

This change is focused on being able to send json (however formatted) and receive spans in json (v1 or v2) or thrift formats.